terraform/roar_variables.tf

```diff
@@ -0,0 +1,11 @@
+variable "mix_validators" {
+  type    = bool
+  default = false
+  description = "Mix validator images for ROAR test"
+}
+
+variable "mix_image_tags" {
```

Comment:
why is it called "tags" here? yet below seems to be "versions" ?

Reply:
following the same convention as `image_tag` --> `image_version` when in the terraform config (e.g. we check if we've passed in a sha256 hash or an explicitly tagged version)